streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractKeyValueStoreTest.java

```diff
@@ -67,6 +69,34 @@ public void after() {
         return result;
     }
 
+    @Test
+    public void testDelete() {
+        store.close();
+
+        final Serializer<String> serializer = new StringSerializer() {
+            @Override
+            public byte[] serialize(final String topic, final String data) {
+                if (data.equals("null")) {
+                    return null;
+                }
+                return super.serialize(topic, data);
+            }
+        };
+
+        context.setValueSerde(Serdes.serdeFrom(serializer, new StringDeserializer()));
+        store = createKeyValueStore(driver.context());
+
+        store.put(0, "zero");
+        store.put(1, "one");
+        store.put(2, "two");
+        store.delete(0);
```

Comment:
Should we add a test for `delete` to make sure it does not call the serializer?

Reply:
Not sure I understand.. do you mean the serializer for the key is not called (which is not the case)? Or the serializer of the value (we do not have value parameter)?

Reply:
I referred to the value serializer.